tasks/zkml-convert-model/main.py

```diff
@@ -0,0 +1,121 @@
+from pathlib import Path
+
+import json
+import asyncio
+import logging
+
+from coretex import currentTaskRun, folder_manager, Model
+
+import ezkl
+import onnx
+import numpy as np
+
+
+def linkFolder(source: Path, destination: Path) -> None:
+    destination.mkdir(exist_ok = True)
+    for item in source.rglob("*"):
+        destItem = destination / item.relative_to(source)
+
+        if item.is_dir():
+            continue
+        elif item.is_file():
```

Comment:
`elif` can be removed as the `if` is already doing `continue` if the criteria is met